README.md

```diff
@@ -6,61 +6,199 @@ NOTE: This repository is a WORK IN PROGRESS.
 
 ---
 
-# Datasets Native To This Package
+# Installation
 
-NOTE: Information. Section of the data. How to instantiate. Reason for being. How it was created.
+Install `forecasttools` via:
 
+```
+pip3 install git+https://github.com/CDCgov/forecasttools-py@main
+```
+
+# Vignettes
+
+* Format Arviz Forecast Output For FluSight Submission (In Progress)
+
+# Datasets
+
+By default, `forecasttools` contains several datasets. These datasets aid with experimentation or are directly necessary to some of `forecasttools` utilities.
 
 ## Location Table
 
+The location table contains abbreviations, codes, and extended names for the US jurisdictions for which the FluSight and COVID forecasting hubs require users to generate forecasts.
+
+Shape: (58, 3)
+
+| location_code | short_name | long_name                   |
+| ---           | ---        | ---                         |
+| str           | str        | str                         |
+|---------------|------------|-----------------------------|
+| US            | US         | United States               |
+| 1             | AL         | Alabama                     |
+| 2             | AK         | Alaska                      |
+| 4             | AZ         | Arizona                     |
+| 5             | AR         | Arkansas                    |
+| 6             | CA         | California                  |
+| 8             | CO         | Colorado                    |
+| 9             | CT         | Connecticut                 |
+| …            | …         | …                          |
+| 56            | WY         | Wyoming                     |
+| 60            | AS         | American Samoa              |
+| 66            | GU         | Guam                        |
+| 69            | MP         | Northern Mariana Islands    |
+| 72            | PR         | Puerto Rico                 |
+| 74            | UM         | U.S. Minor Outlying Islands |
+| 78            | VI         | U.S. Virgin Islands         |
+
```

Comment:
For all these outputs, consider using quarto to generate the readme (ideally in a github action) so it always shows the current table.

Reply:
For this PR or another?

Reply:
Probably another. Please make an issue.

Reply:
See #19